datasets/conv_questions/conv_questions.py

```diff
@@ -0,0 +1,172 @@
+(https://dl.acm.org/citation.cfm?id=3358016).
+"""
+
+
+import json
+import os
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@InProceedings{christmann2019look,
+  title={Look before you hop: Conversational question answering over knowledge graphs using judicious context expansion},
+  author={Christmann, Philipp and Saha Roy, Rishiraj and Abujabal, Abdalghani and Singh, Jyotsna and Weikum, Gerhard},
+  booktitle={Proceedings of the 28th ACM International Conference on Information and Knowledge Management},
+  pages={729--738},
+  year={2019}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+ConvQuestions is the first realistic benchmark for conversational question answering over knowledge graphs.
+It contains 11,200 conversations which can be evaluated over Wikidata. The questions feature a variety of complex
+question phenomena like comparisons, aggregations, compositionality, and temporal reasoning."""
+
+_HOMEPAGE = "https://convex.mpi-inf.mpg.de"
+
+_LICENSE = "MIT License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "http://qa.mpi-inf.mpg.de/convex/"
+_URLs = {
+    "train": _URL + "/ConvQuestions_train.zip",
+    "dev": _URL + "/ConvQuestions_dev.zip",
+    "test": _URL + "/ConvQuestions_test.zip",
+}
+
+
+class ConvQuestions(datasets.GeneratorBasedBuilder):
+    """ConvQuestions is a realistic benchmark for conversational question answering over knowledge graphs."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('conv_questions', 'train')
+    # data = datasets.load_dataset('conv_questions', 'dev')
+    # data = datasets.load_dataset('conv_questions', 'test')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="train", version=VERSION, description="Train set of the benchmark."),
+        datasets.BuilderConfig(name="dev", version=VERSION, description="Validation set of the benchmark."),
+        datasets.BuilderConfig(name="test", version=VERSION, description="Test set of the benchmark."),
+    ]
+
+    DEFAULT_CONFIG_NAME = "train"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
```

Comment:
You can remove all this. We distinguish configurations and splits. Configurations can be used when there are several formats/tasks/processing for you dataset. We use splits for train/test/validation separation. In your case you don't need to have configurations.

To load one split one can simply do
```python
load_dataset("conv_questions", split="train")
```

```suggestion
```